backend/es/es_alias.py

```diff
@@ -51,6 +51,13 @@ def alias(corpus: Corpus, clean=False):
     logger.info('Done updating aliases')
 
 
+def get_current_index_name(corpus: CorpusConfiguration, client) -> str:
+    """get the name of the current corpus' associated index"""
+    alias = corpus.es_alias or corpus.es_index
+    indices = client.indices.get(index="{}".format(alias))
+    return max(sorted(indices.keys()))
```

Comment:
Is this sorting alphabetically? That would mean that if `my_corpus_9` and `my_corpus_10` both exist, you're use `my_corpus_9`, no?